client_app/src/Shifts.js

```diff
@@ -29,5 +26,15 @@ function Shifts(){
      </div>
    )
 }
+export function UpcomingShifts(){
+   const time_now = new Date().getTime()
+   console.log(time_now)
+   const shelters_endpoint = SERVER+'/shifts?filter_start_after='+time_now;
+   return Shifts(shelters_endpoint)
+}
 
-export default Shifts;
+export function PastShifts(){
+   const time_now = new Date().getTime()
+   const shelters_endpoint = SERVER+'/shifts?filter_end_before='+time_now;
+   return Shifts(shelters_endpoint)
+}
```

Comment:
Is this going to be implemented later into the volunteer dashboard? I saw it and didn't know where it was planned to be used in the code.

Reply:
I added it here to demonstrate how past shifts can be retrieved. This could be used in the dashboard later. I imagine volunteers may want to see when and where they have volunteered in the past. No immediate plans to include past shifts in the UI, but it's here if needed.